terraform/modules/happy-service-eks/main.tf

```diff
@@ -85,7 +85,7 @@ resource "kubernetes_deployment" "deployment" {
           }
 
           dynamic "env" {
-            for_each = nonsensitive(var.additional_env_vars)
+            for_each = var.additional_env_vars
```

Comment:
nonsensitive is no longer required because of https://github.com/chanzuckerberg/happy-infra/pull/179